entrypoint.sh

```diff
@@ -395,6 +393,14 @@ if [ "$PROXY" == "on" ]; then
   printf "DONE\n"
 fi
 
+############################################
+# READ FORWARDED PORT
+############################################
+
+if [ "$PORT_FORWARDING" == "true" ]; then
+  sleep 10 && /portforward.sh &
```

Comment:
Instead of sleeping 10 seconds, get your public IP before using the VPN (using `wget -qO- https://ipinfo.io/ip`) and check for an IP change every 2 seconds? (not too short or it'll be rate limited)

That will work better on very slow connections and is more resilient in my opinion.

Reply:
I'll can try this but it will complicate the script and likely make a breaking change with the previous behavior, because the timing logic will have to be included in `porforward.sh`.

Reply:
Yes you are right, let's keep it that way for this PR, I'll try doing it in a week or so.